src/worker/pbs_info.rs

```diff
@@ -0,0 +1,52 @@
+use std::process::Command;
+use std::{str, env};
+use std::time::Duration;
+use serde_json::Value;
+
+type TimeNumberFormat = u64;
+
+
+fn process_time_format(raw_time: String) -> TimeNumberFormat {
```

Comment:
You can just receive `&str` instead of `String`. `String` is an owned type which allocates on the heap, it shouldn't be needed here.